src/main/java/io/crate/jmx/CrateCollector.java

```diff
@@ -198,6 +200,22 @@ private void recordNumericMBeanValue(LinkedHashMap<String, String> beanPropertie
         }
     }
 
+    private void recordStringMBeanValue(String attrName,
+                                        String mBeanName,
+                                        String value) {
+        Recorder recorder = RecorderRegistry.get(mBeanName);
+        if (recorder != null) {
+            boolean supportedAttribute = recorder.recordBean(CRATE_DOMAIN_REPLACEMENT, attrName, value, this::addSample);
+            if (supportedAttribute == false) {
```

Comment:
In which case is a string value supported? Couldn't we just always ignore these?

Reply:
right, will remove support for it